test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_smtrcomp.py

```diff
@@ -309,6 +309,46 @@ def test_training(self, sagemaker_session, ecr_image, framework_version, instanc
         captured = capsys.readouterr()
         _assert_training_compiler_invoked(captured)
 
+    @pytest.mark.model('distilbert')
+    def test_BYOC_training(self, sagemaker_session, ecr_image, framework_version, instance_type, instance_count,tmpdir, capsys):
+        source_path = os.path.join(resource_path, 'mlm')
+        estimator = TensorFlow(
+            entry_point="run_mlm.py",
+            source_dir=source_path,
+            role='SageMakerRole',
+            instance_type=instance_type,
+            instance_count=instance_count,
+            sagemaker_session=sagemaker_session,
+            image_uri=ecr_image,
+            framework_version=framework_version,
+            disable_profiler=True,
+            debugger_hook_config=False,
+            py_version="py38",
+            volume_size=500,
+            model_dir=False,
+            hyperparameters={
+                TrainingCompilerConfig.HP_ENABLE_COMPILER : True,
+                "model_name_or_path": "distilbert-base-uncased",
+                "max_seq_length": 128,
+                "dataset_name": "wikitext",
+                "dataset_config_name": "wikitext-2-raw-v1",
+                "fp16": 1,
+                "num_train_epochs": 1,
```

Comment:
also add `max_steps=3` . Training for an entire epoch with a real dataset for an integration test is a waste of resources.